examples/with-msw/pages/index.js

```diff
@@ -1,15 +1,25 @@
 import { useState } from 'react'
 
-export default function Home({ book }) {
+export default function Home({ book, error }) {
   const [reviews, setReviews] = useState(null)
 
   const handleGetReviews = () => {
     // Client-side request are mocked by `mocks/browser.js`.
+    // This request will fail in production, as MSW is only run in development.
```

Comment:
This statement is not entirely true. It's run everywhere _except_ production. Perhaps, we can drop this line altogether, and only mention that in production this request would hit the actual backend?